tests/test_exposure.py

```diff
@@ -648,16 +648,16 @@ def testImageSlices(self):
         """Test image slicing, which generate sub-images using Box2I under the covers"""
         exp = afwImage.ExposureF(10, 20)
         mi = exp.getMaskedImage()
-        mi[9, 19] = 10
+        mi.image[9, 19, afwImage.PARENT] = 10
```

Comment:
see comment in `test_background.py`. There's no need to include the origin since `PARENT` is the default value.